toucan_connectors/aircall/constants.py

```diff
@@ -0,0 +1,71 @@
+"""File containing constants for AirCall connector"""
+
+COLUMN_DICTIONARY = {
+    'calls': [
+        'id',
+        'direction',
+        'duration',
+        'answered_at',
+        'ended_at',
+        'raw_digits',
+        'user_id',
+        'tags',
+        'user_name',
+        'team',
+        'day',
+    ],
+    'tags': ['id', 'name', 'color', 'description'],
+    'users': ['team', 'user_id', 'user_name', 'user_created_at'],
+}
+
+FILTER_DICTIONARY = {
```

Comment:
Genuine question, since we're writing some python here, is there a real benefit in writing jq filters rather than standard, tested python code?